src/main/java/uk/nhs/hee/tis/trainee/forms/migration/SortWorkPlacements.java

```diff
@@ -60,7 +60,8 @@ public SortWorkPlacements(MongoTemplate mongoTemplate, FormRPartBService formSer
    */
   @Execution
   public void migrate() {
-    Comparator<Work> compareEndDates = Comparator.comparing(Work::getEndDate).reversed();
+    Comparator<Work> compareEndDates = Comparator.comparing(Work::getEndDate,
+        Comparator.nullsLast(Comparator.naturalOrder())).reversed();
```

Comment:
This is equivalent to `Comparator.nullsFirst(Comparator.reverseOrder()))`.

Did you want to preserve/make clear that it is ordered in a particular way and then reversed or is the alternative a more accurate way of describing the intended ordering?